sympy/solvers/ode.py

```diff
@@ -6468,7 +6468,7 @@ def _linear_2eq_order1_type1(x, y, t, r):
 
     """
     l = Dummy('l')
-    C1, C2, C3, C4 = symbols('C1:5')
+    C1, C2, C3, C4 = get_numbered_constants(Eq(x(t)), num=4)
```

Comment:
why `Eq(x(t))` here?  Should it have something to do with `y` too?  Will this search the whole given DE?